client/src/use/guiElements.js

```diff
@@ -1,8 +1,23 @@
-import { computed } from "vue"
+import { computed, ref, watch } from "vue"
 import { useQuasar } from "quasar"
 import { useI18n } from "vue-i18n"
 import { useCurrentUser } from "./user"
 
+export function useDarkMode() {
+  const $q = useQuasar()
+  const darkModeStatus = ref($q.dark.isActive)
+  watch(
+    () => $q.dark.isActive,
+    () => {
+      darkModeStatus.value = $q.dark.isActive
+    },
+  )
+  function toggleDarkMode() {
+    $q.dark.toggle()
+  }
+  return { darkModeStatus, toggleDarkMode }
+}
+
```

Comment:
So, pardon if I'm missing something, but why wouldn't we just use `const { dark: { toggle, isActive} = useQuasar()` in the component rather than wrapping this in our own composable?